app/src/main/res/layout/fragment_user_name_test_and_set.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".userProfileCreation.UserNameTestAndSetFragment"
+    android:id="@+id/userName_frame"
+    android:background="@drawable/image_mountain_background"
+    >
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:layout_gravity="center">
+        <TextView
+            android:id="@+id/select_userName_title_userProfileCreation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:padding="4dp"
+            android:text="@string/please_enter_a_user_name"
+            android:textIsSelectable="false"
+            android:textSize="28sp"
+            android:textColor="@color/design_default_color_surface"
+            android:textAlignment="center"
+            android:textStyle="bold"/>
+        <TextView
+            android:id="@+id/select_userName_description_userProfileCreation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:padding="4dp"
+            android:text="@string/you_can_click_on_the_check_availability_button_to_check_if_your_proposition_is_available_after_you_click_on_the_validate_button_your_user_name_is_fixed"
+            android:textIsSelectable="false"
+            android:textSize="15sp"
+            android:textColor="@color/design_default_color_surface"
+            android:background="@color/cardview_shadow_start_color"
+            android:justificationMode="inter_word"/>
+
+        <Space
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_weight="1"/>
+
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="40sp"
+            android:id="@+id/input_userName_text_UserProfileCreation"
+            android:hint="@string/user_name"
+            android:background="@color/design_default_color_surface"
+            />
```

Comment:
You could add these so that it stays on 1 line
`android:maxLines="1"`
`android:lines="1"`
`android:singleLine="true"`